src/en/models-config.md

```diff
@@ -1,72 +1,135 @@
 Title: General configuration options  
-Commands: get-env, set-env  
+TODO: check options haven't changed
 
-# General configuration options
+# Configuring models
 
 There may be times, particularly when dealing with complex cloud environments
 or running multiple instances of Juju, where it would be useful to control
-further aspects of the Juju environment. Many options can be changed by adding
-extra key:value pairs to the environments.yaml file which Juju uses to initiate
-environments. Options which are specific to certain cloud providers are
-detailed in the individual configuration pages, but there are additional
-settings which can be applied to any environment. Some of these may be useful
-for particular use cases as outlined in the sections below, or you can
-[skip to the table](#alphabetical-list-of-general-configuration-values) of all
-the values.
+further aspects of the models you create. 
 
 
 ## Getting and setting individual values
 
-You can display the environment settings of a running Juju environment by
+You can display the current model settings by
 running the command:
 
 ```bash
-juju get-env
+juju get-model-config
 ```
 
 This will include all the currently set options - whether they were set
-by the individual configuration in the environments.yaml file, set specifically
-by you or inherited as a default value. 
+by you, inherited as a default value or dynamically set by Juju. 
 
-You may also set individual values for the current environment using the
-corresponding ```juju set-env``` command, and providing a key=value pair:
+You may also set values for the current environment using the
+corresponding ```juju set-model-config``` command, and providing a key=value 
+pair:
 
 ```bash
-juju set-env noproxy=jujucharms.com
+juju set-model-config noproxy=jujucharms.com
+```
+It is possible to specify a list of space-delimited key-value pairs to set more
+than one configuration at a time:
+  
+```bash
+juju set-model-config test-mode=true enable-os-upgrade=false
```

Comment:
Not sure what you mean. If you mean, do we want a scary error message every time we add some key that isn't there by default - see https://bugs.launchpad.net/juju-core/+bug/1413653